public/js/components/Editor.css

```diff
@@ -10,7 +10,7 @@
  */
 .editor-wrapper {
   position: absolute;
-  height: calc(100% - 30px);
+  height: calc(100% - 31px);
```

Comment:
I don't remember if it's a chrome thing, there were certainly chrome things with codemirror expanding.